Cargo.toml

```diff
@@ -261,13 +261,13 @@ sources-journald = []
 sources-kafka = []
 sources-logplex = ["warp", "sources-tls"]
 sources-prometheus = []
-sources-socket = ["bytesize", "listenfd", "tokio/uds", "tokio-util/udp", "sources-tls"]
+sources-socket = ["bytesize", "listenfd", "tokio-util/udp", "sources-tls"]
 sources-splunk_hec = ["bytesize", "warp", "sources-tls"]
 sources-statsd = ["tokio-util/udp"]
 sources-stdin = ["bytesize"]
-sources-syslog = ["sources-socket", "syslog_loose"]
-sources-tls = ["sources-http", "sources-logplex", "sources-socket", "sources-splunk_hec"]
-sources-vector = ["sources-socket"]
+sources-syslog = ["bytesize", "listenfd", "tokio-util/udp", "sources-tls", "syslog_loose"]
+sources-tls = []
```

Comment:
Looks like this solve my question https://github.com/timberio/vector/pull/3188#issuecomment-663712811

Reply:
Yep.